src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -99,6 +99,7 @@ class Condition(StrEnum):
     Condition.EXISTING_HIGH_PRIORITY_ISSUE,
     Condition.ISSUE_CATEGORY,
     Condition.ISSUE_RESOLUTION_CHANGE,
+    Condition.ISSUE_PRIORITY_EQUALS,
```

Comment:
👍 to this - i just noticed the `Condition.EVERY_EVENT` too -- fyi, if we want the Automation UI to have "Every Occurrence" kind of concept in it, we could reuse that condition.

Reply:
good point, i'll remove EVERY_EVENT from the ignore list 🫡